pkg/destroy/baremetal/baremetal.go

```diff
@@ -1,3 +1,5 @@
+// +build ironic
```

Comment:
I'm missing something here - why is the build tag "ironic" and not "baremetal"?

Reply:
It probably should be baremetal, but the tag we use is ironic.

Reply:
Ah I see it's already used for some of the terraform bits - it seems weird to expose an internal implementation detail to those building, and it's inconsistent with other platforms, so perhaps we can do a follow-up which fixes all the tags to match the platform name?